MantidQt/CustomInterfaces/src/Reflectometry/ReflMainViewPresenter.cpp

```diff
@@ -105,6 +104,8 @@ void ReflMainViewPresenter::notify(IReflPresenter::Flag flag) {
 /** Pushes the list of commands (actions) */
 void ReflMainViewPresenter::pushCommands() {
 
+  m_view->clearCommands();
```

Comment:
I added a couple of lines to test that this method gets called: [here](https://github.com/mantidproject/mantid/blob/14921_Refactor_ReflMainView_and_ReflMainViewPresenter/MantidQt/CustomInterfaces/test/ReflMainViewPresenterTest.h#L71) and [here](https://github.com/mantidproject/mantid/blob/14921_Refactor_ReflMainView_and_ReflMainViewPresenter/MantidQt/CustomInterfaces/test/ReflMainViewPresenterTest.h#L48), the idea was to test that this method gets called when the presenter is constructed and when something changes in the ADS, but I may be missing something.. if so, let me know